packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -0,0 +1,112 @@
+const ACCOUNTS_DOCTYPE = 'io.cozy.accounts'
+const PERMISSIONS_DOCTYPE = 'io.cozy.permissions'
+
+/**
+ * Performs call to create an io.cozy.accounts document
+ * @param  {Object}  client     CozyClient
+ * @param  {Object}  attributes Account attributes
+ * @return {Object}             The io.cozy.accounts document just created
+ */
+const createAccount = async (client, attributes) => {
+  const { data } = await client.create(ACCOUNTS_DOCTYPE, attributes)
+  return data
+}
+
+/**
+ * Performs a query to find the given account
+ * @param  {Object}  client CozyClient
+ * @param  {string}  id     io.cozy.accounts document's id
+ * @return {Object}         Retrieved account
+ */
+const findAccount = async (client, id) => {
+  const { data } = await client.query({ ...client.where(ACCOUNTS_DOCTYPE), id })
+  return data[0]
```

Comment:
No need to be defensive?

Reply:
This function has changed and it is now relying on `client.get`.